src/Scripts/DockerLocalBuild.ps1

```diff
@@ -41,17 +55,39 @@ param
 
 $ErrorActionPreference = "Stop"
 
+if ($LocalDockerImage) {
+    & $PSScriptRoot\DockerLocalImage -EnlistmentRoot $EnlistmentRoot -CLib $CLib
+    if (-not $?) {
+        write-error "Error creating docker image"
+        exit 1
+    }
+}
+
 if ($CLib -eq "gnu")
 {
-    $BaseImage = "proddiagbuild.azurecr.io/clrie-build-ubuntu:latest"
+    if ($LocalDockerImage)
+    {
+        $BaseImage="clrielocal:gnu"
```

Comment:
>$BaseImage="clrielocal:gnu" [](http://example.com/codeflow?start=8&length=27)

Since the DockerLocalImage.ps1 script generates this, you could have the script `Write-Output $imageName` (you'll want to use Write-Verbose or Write-Host for logging) and store that value into $BaseImage.